cabal-install-solver/ChangeLog.md

```diff
@@ -1 +1 @@
-Please see https://github.com/haskell/cabal/blob/master/release-notes/cabal-install-3.10.3.0.md
```

Comment:
this shouldn’t be in cabal-install-solver, but in `Cabal-syntax`

(although I guess bumping `cabal-install-solver` is good too)

Reply:
Right, was trying to catch all at once because, unlike releasing artifacts, there's no distinction in the changelogs step